lib/igniter/code/common.ex

```diff
@@ -226,20 +230,29 @@ defmodule Igniter.Code.Common do
   \"\"\"
   ```
   """
-  @spec add_code(Zipper.t(), String.t() | Macro.t(), :after | :before) :: Zipper.t()
-  def add_code(zipper, new_code, placement \\ :after)
+  @spec add_code(Zipper.t(), String.t() | Macro.t(), add_code_opts() | atom()) :: Zipper.t()
+  def add_code(zipper, new_code, opts \\ [placement: :after, expand_env?: true])
```

Comment:
We need to specify the defaults when they're extracted anyways (which you're already doing), so this might as well be an empty list to avoid the defaults ever getting out-of-sync.

```suggestion
  def add_code(zipper, new_code, opts \\ [])
```

Reply:
Good call! fixed in [`71f1574` (#151)](https://github.com/ash-project/igniter/pull/151/commits/71f1574077e7374056352ff1528a1d8ed449cbd3)